src/main/java/uk/gov/justice/digital/delius/data/api/deliusapi/NewRelease.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.delius.data.api.deliusapi;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@NoArgsConstructor
```

Comment:
I m not sure you need this. Doe the NewRelase bein construct even with an empty constructor ? Or Actualrelease , releasetype and institution are mandatory fields ? If is the case. Remove annotation , make the fields final.

Reply:
yup sorted that.